pegasus/migrations/83_create_channel_table_metadata.rb

```diff
@@ -0,0 +1,20 @@
+Sequel.migration do
+  up do
+    create_table?(:channel_table_metadata, charset: 'utf8') do
+      primary_key :id, unsigned: true, null: false
+      foreign_key :app_id, index: true, null: false
+      foreign_key :storage_id, index: true
+      String :table_name, size: 50, index: true, null: false
+
+      Text :column_list
+
+      DateTime :updated_at, null: false
+
+      index [:channel_id,:storage_id,:table_name], unique: true
+    end
+  end
+
+  down do
+    drop_table?(:channel_table_metadata)
+  end
```

Comment:
Could/should this instead be `unless CDO.use_dynamo_tables` ? Or will we not have access to CDO config here?